tests/requirements-relaxed.txt

```diff
@@ -92,7 +92,7 @@ pyyaml==6.0.1
     #   antsibull-docs
 requests==2.31.0
     # via sphinx
-resolvelib==1.0.1
+resolvelib==0.8.1
```

Comment:
This was also automatically fixed by pip-compile. `tests/requirements-base.in` has 

    resolvelib >= 0.5.3, < 0.9.0 # used by ansible-core

so this never should have been bumped to 1.0.1.

Reply:
@gotmax23 hmm. I can't tell from commit history but guessing this was incorrectly bumped when we set up the relaxed requirements on stable-2.14. nice catch here.